frontend/src/actions/appActions.js

```diff
@@ -0,0 +1,5 @@
+import * as types from './actionsTypes';
+
+export function changeView(appView){
```

Comment:
Could you name the file with reference to the function of the actions? i.e changeViewAction.js